[ECS] 4.0alpha1 bug

```
 [ERROR] Argument 1 passed to Symplify\EasyCodingStandard\Yaml\CheckerTolerantYamlFileLoader::isReservedKey() must be of
         the type string, integer given, called in
         .../vendor/symplify/easy-coding-standard/src/Yaml/CheckerTolerantYamlFileLoader.php on line 114   
```

Caused by [this line](https://github.com/Symplify/Symplify/blob/master/packages/EasyCodingStandard/config/symfony.yml#L29) - `$key` is `0` and `$value` is `'curly_brace_block'`

--- a/packages/EasyCodingStandard/src/Yaml/CheckerTolerantYamlFileLoader.php
+++ b/packages/EasyCodingStandard/src/Yaml/CheckerTolerantYamlFileLoader.php
@@ -152,8 +152,15 @@ final class CheckerTolerantYamlFileLoader extends YamlFileLoader
         return Strings::replace($value, '#@#', '@@');
     }
 
-    private function isReservedKey(string $key): bool
+    /**
+     * @param string|int $key
+     */
+    private function isReservedKey($key): bool
     {
+        if (is_int($key)) {
+            return false;
+        }
+
         return in_array($key, $this->getServiceKeywords(), true);
     }
 

--- a/packages/EasyCodingStandard/src/Yaml/CheckerTolerantYamlFileLoader.php
+++ b/packages/EasyCodingStandard/src/Yaml/CheckerTolerantYamlFileLoader.php
@@ -153,7 +153,7 @@ final class CheckerTolerantYamlFileLoader extends YamlFileLoader
     }
 
     /**
-     * @param string|int $key
+     * @param string|int|bool $key
      */
     private function isReservedKey($key): bool
     {

--- a/packages/EasyCodingStandard/src/Yaml/CheckerTolerantYamlFileLoader.php
+++ b/packages/EasyCodingStandard/src/Yaml/CheckerTolerantYamlFileLoader.php
@@ -157,7 +157,7 @@ final class CheckerTolerantYamlFileLoader extends YamlFileLoader
      */
     private function isReservedKey($key): bool
     {
-        if (is_int($key)) {
+        if (! is_string($key)) {
             return false;
         }
 

